styles/hobby-blog.css

```diff
@@ -0,0 +1,81 @@
+html {
```

Comment:
I think this file could be DRYed up a little bit -- a lot of things seems to overlap.